k8s/ci/test-script.sh

```diff
@@ -68,7 +69,19 @@ sleep 10
 #echo "------------------ Deploy Pre-release features ---------------------------"
 #kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-pre-release-features.yaml
 
-echo "------------------------ Create sparse storagepoolclaim --------------- "
+echo "------------------------ Create block device claim --------------- "
+##TODO: Remove below snippet once manual BDC is supported to create cStor pools
+#### Get name of blockdevice to claim the blockdevice ####
+block_device_name=$(kubectl get blockdevice -n ${NS} -o jsonpath='{.items[*].metadata.name}')
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/blockdeviceclaim.yaml -O temp_bdc.yaml
+sed "s|blockdevice-1234|$block_device_name|g" temp_bdc.yaml > bdc.yaml
+kubectl apply -f bdc.yaml
+sleep 15
+block_device_status=$(kubectl get blockdevice -n ${NS} -o jsonpath='{.items[*].status.claimState}')
+echo "BlockDevice: ${block_device_name} Status: ${block_device_status}"
```

Comment:
IMO, giving the name as `Status` can be confusing. As the status field has two values: `.status.claimState` and `.status.State`.

Reply:
Updated